content/en/docs/concepts/windows/intro.md

```diff
@@ -341,7 +341,7 @@ On Windows nodes, strict compatibility rules apply where the host OS version mus
 match the container base image OS version. Only Windows containers with a container
 operating system of Windows Server 2019 are fully supported.
 
-For Kubernetes v{{< skew currentVersion >}}, operating system compatibility for Windows nodes (and Pods)
+For Kubernetes v{{< skew currentPatchVersion >}}, operating system compatibility for Windows nodes (and Pods)
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.